packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -186,13 +209,13 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
         <Modal isOpen={true} isModeless={true} dragOptions={DRAG_OPTIONS} styles={floatingLocalVideoModalStyle}>
           {localParticipant && defaultOnRenderLocalVideoTile}
         </Modal>
-        <GridLayout styles={styles}>{defaultOnRenderRemoteParticipants}</GridLayout>
+        <GridLayout styles={styles ?? {}}>{defaultOnRenderRemoteParticipants}</GridLayout>
```

Comment:
Will this create spurious renders for `GridLayout` when `styles` is `undefined`? This creates a new object each time.

Reply:
It should not. But I get nervous around such topics and will change it. :D